build.gradle

```diff
@@ -31,8 +31,8 @@ repositories {
 
 dependencies {
     compile "org.scala-lang:scala-library:${scalaLibraryVersion}"
+    compile "io.circe:circe-core_${scalaVersion}:0.11.2"
```

Comment:
If you need this for tests, please add this dependency as `compileOnly` and `testCompile`.

Reply:
OK, changed